app/assets/stylesheets/pages/contact.scss

```diff
@@ -0,0 +1,49 @@
+/*Contact Styles */
+
+.contact {
+
+	.questions {
+		color: $orange;
+		font-size: $font-size-l;
+		font-weight: $font-weight-large;
+		text-align: center;
+	}
+
+	.input-field {
+	    padding: 12px 20px;
+	    margin: 12px 0;
+	    box-sizing: border-box;
+	    border-style: solid;
+	    border-width: 1px;
+    	border-color: $grey;
+	}
+
+	.half-width-input {
+		width: 48%;
+	}
+
+	.full-width-input {
+		width: 100%;
+	}
+
+	.last-name-input {
+		margin-left: 4%;
```

Comment:
You should try to avoid using percentages for margins if possible. It can be unclear what it is a percentage of.